play-json/shared/src/main/scala/play/api/libs/json/Json.scala

```diff
@@ -248,6 +248,24 @@ object Json extends JsonFacade {
    */
   def reads[A]: Reads[A] = macro JsMacroImpl.implicitConfigReadsImpl[A]
 
+  /**
+   * Creates a `Reads[A]`, if `A` is a ValueClass,
+   * by resolving at compile-time the required implicits.
+   *
+   * $macroWarning
+   *
+   * $macroTypeParam
+   *
+   * {{{
+   * import play.api.libs.json.{ Json, Reads }
+   *
+   * final class IdText(val value: String) extends AnyVal
+   *
+   * val r: Reads[IdText] = Json.valueReads
+   * }}}
+   */
+  def valueReads[A]: Reads[A] = macro JsMacroImpl.implicitConfigValueReads[A]
```

Comment:
@richdougherty If we intend that to be only in 2.7.0, I think it may be possible to update the `def reads` to have 2 unambiguous variants (able to infer whether to call the ValueClass macro or not), only using `reads[T]` at the calling site.

Reply:
It would be nice to get this into 2.6.x, so maybe we keep it like this and we can improve upon `reads` in 2.7?